src/main/java/uk/gov/digital/ho/hocs/audit/export/infoclient/ExportViewConstants.java

```diff
@@ -2,11 +2,15 @@
 
 public interface ExportViewConstants {
 
-        String FIELD_ADAPTER_HIDDEN = "hidden";
-        String FIELD_ADAPTER_USER_EMAIL = "userEmailAdapter";
-        String FIELD_ADAPTER_USERNAME = "usernameAdapter";
-        String FIELD_ADAPTER_FIRST_AND_LAST_NAME = "userFirstAndLastName";
-        String FIELD_ADAPTER_TEAM_NAME = "teamNameAdapter";
-        String FIELD_ADAPTER_UNIT_NAME = "unitNameAdapter";
-        String FIELD_ADAPTER_TOPIC_NAME = "topicNameAdapter";
+    // identifies malformed dates with groups for extraction
+    String GROUPED_DATE_REGEX = "^0*(\\d{4})\\-0*(0[0-9]|1[0-2])\\-0*(0[1-9]|[12][0-9]|3[01])$";
```

Comment:
```suggestion
    String GROUPED_DATE_REGEX = "^0*(\\d{4})\\-0*(0[0-9]|1[0-2])\\-0*([012][0-9]|3[01])$";
```
We can simplify the regex a little further...